source/documentation/runbooks/delayed-job-failures.html.md.erb

```diff
@@ -88,7 +88,53 @@ kubectl rollout restart deployments -n formbuilder-platform-test-dev fb-user-fil
 
 It is at least worth rolling the filestore pods. It may also be worth rolling the submitter pods too. If that still doesn't solve the problem then it might need a manual reply and/or further investigation.
 
-### Replaying failed jobs
+### Removing a problem attachment
+Sometimes attachments prevent a submission from being processed, most commonly this is due to an attachment with a mime type that is not recognised by our platform.
+
+Unfortunately, we will need to remove the problem attachment to replay it, and inform the form owner of this.
+
+Before removing the attachment, ensure you have the following information:
+
+- Some way to identify the submission (Submission ID or Case Number)
+- File name for the failed attachment
+- The form/service that was used to submit the form
+
+Once we have this information, we can remove the attachment from the submission.
+
+Note the following examples assume the submission is from the legacy platform, please use `decrypted_submission` if the submission was form the MoJ Forms platform.
+
+Save the decrypted submission to another variable:
+
+~~~~~~~~
+updated_submission = Submission.find(submission_id).decrypted_payload
+~~~~~~~~
+
+Find the problematic attachment in the 'attachments' key:
+
+~~~~~~~~
+updated_submission['attachments']
+~~~~~~~~
+
+Remove the attachment accordingly.
+
+You can double check the attachments:
+
+~~~~~~~~
+updated_submission['attachments']
+~~~~~~~~
+
+Once you are happy that you have removed the problem attachment, you will need to update and re-encrypt the submission.
+
+~~~~~~~~
+submission.update(payload: EncryptionService.new.encrypt(updated_submission))
```

Comment:
Probably need to include the MOJ forms (v2) way of doing things here?  Or just replace the re-encrypting bit with a reference to the re-encrypting bit below?

Reply:
good point, I've updated with a link to the re-encrypting bit below